### DISCUSSION TITLE
Version 3.2.0

### DISCUSSION THREAD

**Discussion by System0X** [confluentinc/parallel-consumer#149] [Q&A] [answered]
Hi, the version 3.2.0 is not in the central maven repository. Is there a problem?

**Comment by astubbs** [comment, marked as answer]
Oh my! Yes there was a problem. The release build failed, and I didn't notice. Thank you for asking! I will run the release build again now. (Apologies for delay, I've been off sick).

**Reply by astubbs** [reply]
Ok, build completed - artefacts should be up in 15 minutes https://repo1.maven.org/maven2/io/confluent/parallelconsumer/parallel-consumer-parent/

**Reply by astubbs** [reply]
Fixed.
